src/internal/_xwrap.js

```diff
@@ -0,0 +1,20 @@
+var identity = require('../identity');
+
+module.exports = (function() {
+    function _xwrap(fn) {
+        return new XWrap(fn);
+    }
+
+    function XWrap(fn) {
+        this.f = fn;
+    }
+    XWrap.prototype.init = function() {
+        throw new Error('init not implemented on XWrap');
+    };
+    XWrap.prototype.result = identity;
```

Comment:
We probably don't want `identity` with its `R.__` handling. We could define `_identity` or define it inline.